components/rhs_comment/rhs_comment.jsx

```diff
@@ -361,6 +368,22 @@ class RhsComment extends React.PureComponent {
                         />
                     </Badge>
                 );
+            } else if (isSystemMessage && this.props.isBot) {
+                userProfile = (
+                    <UserProfile
+                        userId={post.user_id}
+                        hideStatus={true}
+                    />
+                );
+
+                visibleMessage = (
+                    <span className='post__visibility'>
+                        <FormattedMessage
+                            id='post_info.message.visible'
+                            defaultMessage='(Only visible to you)'
+                        />
+                    </span>
```

Comment:
Is this shown to only ephemeral bot posts? Or to all?

Reply:
It is shown to all bot posts